spring-cloud-starter-stream-source-load-generator/pom.xml

```diff
@@ -22,6 +22,11 @@
 			<groupId>org.springframework.cloud.stream.app</groupId>
 			<artifactId>app-starters-test-support</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud.stream.app</groupId>
+			<artifactId>app-starters-postprocessor-common</artifactId>
+			<version>2.0.0.BUILD-SNAPSHOT</version>
```

Comment:
The versions management should go to the `*-dependencies` module. In other words to the BOM.

Reply:
ok the version tag is removed and i created a PR to update core at: https://github.com/spring-cloud-stream-app-starters/core/pull/26